### PR TITLE
Fixed some descriptions in backend

### PR DESCRIPTION
**Pull request by ynqa**
Fixed some descriptions in backend.

### CODE REVIEW

File: keras/backend/tensorflow_backend.py [hunk 1]
```diff
@@ -3346,7 +3344,7 @@ def bias_add(x, bias, data_format=None):
         x: Tensor or variable.
         bias: Bias tensor to add.
         data_format: Data format for 3D, 4D or 5D tensors:
-            one of "channels_first", "channels_last".
+            one of `channels_first`, `channels_last`.
```
Comment (fchollet) [inline keras/backend/tensorflow_backend.py]: These are strings, not code keywords

File: keras/backend/tensorflow_backend.py [hunk 2]
```diff
@@ -856,7 +856,7 @@ def update(x, new_x):
     """Update the value of `x` to `new_x`.
 
     # Arguments
-        x: A Variable.
+        x: A variable.
```
Comment (fchollet) [inline keras/backend/tensorflow_backend.py]: In all of these cases, we refer to the `Variable` class.

File: keras/backend/tensorflow_backend.py (hunk 1 above)
Comment (ynqa) [inline keras/backend/tensorflow_backend.py]: okay, I'll rebase them.

File: keras/backend/tensorflow_backend.py (hunk 2 above)
Comment (ynqa) [inline keras/backend/tensorflow_backend.py]: I see. I wonder that it should attach `` ` `` symbols to Variable for understandability.
So I suggest to change to `Variable` in this case.

Comment (fchollet) [inline keras/backend/tensorflow_backend.py]: Sure.

**Comment by ynqa** [comment]
Fixed items pointed out.

File: keras/backend/tensorflow_backend.py [hunk 3]
```diff
@@ -2086,11 +2087,8 @@ def one_hot(indices, num_classes):
         num_classes: Integer, number of classes to consider.
 
     # Returns
-        (n + 1)D one hot representation of the input
+        A tensor has (n + 1)D one hot representation of the input
```
Comment (fchollet) [inline keras/backend/tensorflow_backend.py]: ?

File: keras/backend/tensorflow_backend.py [hunk 4]
```diff
@@ -882,7 +882,7 @@ def update_sub(x, decrement):
     """Update the value of `x` by subtracting `decrement`.
 
     # Arguments
-        x: A Variable.
+        x: A variable.
```
Comment (fchollet) [inline keras/backend/tensorflow_backend.py]: Please change these to be consistent with the above

File: keras/backend/tensorflow_backend.py [hunk 5]
```diff
@@ -895,12 +895,13 @@ def moving_average_update(x, value, momentum):
     """Compute the moving average of a variable.
 
     # Arguments
-        x: A Variable.
-        value: A tensor with the same shape as `variable`.
+        x: A variable.
```
Comment (fchollet) [inline keras/backend/tensorflow_backend.py]: Same

File: keras/backend/tensorflow_backend.py (hunk 3 above)
Comment (ynqa) [inline keras/backend/tensorflow_backend.py]: There are two `Returns`:
```
# Returns
- (n + 1)D one hot representation of the input
# Returns		
-  The one-hot tensor.
```

So I summarized in one.

File: keras/backend/tensorflow_backend.py (hunk 4 above)
Comment (ynqa) [inline keras/backend/tensorflow_backend.py]: oops, I forgot, sorry.

File: keras/backend/tensorflow_backend.py (hunk 3 above)
Comment (fchollet) [inline keras/backend/tensorflow_backend.py]: There's probably a typo in the sentence.

Comment (ynqa) [inline keras/backend/tensorflow_backend.py]: In that case, I'm going to pick up only `(n + 1)D one hot representation of the input...`.

**Comment by fchollet** [comment]
LGTM

**Pull request merged**